docs/helm-chart-submission.md

```diff
@@ -16,7 +21,39 @@ The following options are available for submitting a chart for Red Hat OpenShift
 
 > **_NOTE:_**  It is recommended when submitting a chart to submit chart source over a tarball. 
 
+> **_NOTE:_**  When submitting a Verification report only the report must be generated using the public url for the chart. 
+
 For more information on the submission process, see: [OpenShift Helm Charts Repository documentation](https://github.com/openshift-helm-charts/charts/blob/main/docs/README.md).
 
 For troubleshooting report related submission failures see: [Troubleshooting](./helm-chart-troubleshooting.md)
- 
\ No newline at end of file
+
+## Provider controlled delivery
+
+By default a submitted chart will be made available in the OpenShift Helm Chart Catalogue on successful certification. In some cases this is undesirable and can be prevented based on two conditions:
```

Comment:
By default, a submitted chart will be made available in the OpenShift Helm Chart Catalog on successful certification. In some cases this is undesirable and can be prevented based on the following conditions: